pandera/backends/pyspark/builtin_checks.py

```diff
@@ -37,14 +42,21 @@
         ALL_NUMERIC_TYPE, ALL_DATE_TYPE, STRING_TYPE, BINARY_TYPE, BOLEAN_TYPE
     )
 )
-def equal_to(data: PysparkDataframeColumnObject, value: Any) -> bool:
+@builtin_check_validation_mode()
+def equal_to(
+    data: PysparkDataframeColumnObject,
+    value: Any,
+    should_validate_full_table: bool,
```

Comment:
instead of passing this in as an argument, you can use `pandera.config.get_config_context` to get the `full_table_validation` configuration value. This is so that the API for each check is consistent across the different backends.

Reply:
Thanks @cosmicBboy. Will make the recommended change. 

Also, is there a way that you can suggest to keep the `PANDERA_FULL_TABLE_VALIDATION` config value to be `False` when the backend is pyspark and `True` when the backend is pandas? Did not find a good way to do this, hence asking for a suggestion 😅.

Reply:
You can use the `config_context` context manager in the validate methods for each backend to control this behavior: https://github.com/unionai-oss/pandera/blob/main/pandera/config.py#L71